infra/frontend/app-config/env-config/variables.tf

```diff
@@ -19,3 +19,9 @@ variable "has_database" {
 variable "has_incident_management_service" {
   type = bool
 }
+
+variable "domain" {
```

Comment:
We have it here already! So maybe let's use this one instead in app service, up to you.